client/src/actions/index.ts

```diff
@@ -137,10 +141,9 @@ const doInitialDataLoad = () =>
       const layoutSchema = schema?.schema?.layout?.obs ?? [];
       if (
         defaultEmbedding &&
-        // eslint-disable-next-line @typescript-eslint/no-explicit-any --- FIXME: disabled temporarily on migrate to TS.
-        layoutSchema.some((s: any) => s.name === defaultEmbedding)
+        layoutSchema.some((s: EmbeddingSchema) => s.name === defaultEmbedding)
       ) {
-        dispatch(embActions.layoutChoiceAction(defaultEmbedding));
+        embActions.layoutChoiceAction(defaultEmbedding);
```

Comment:
@seve I think we need to dispatch `embActions.layoutChoiceAction(defaultEmbedding);`, since it's a thunk?

Reply:
Ah I thought I found a bug, didn't see returning function because of lack of `return`